### PR TITLE
ROX-13646: Add logs for port-forwarding in E2E tests

### PR DESCRIPTION
**Pull request by kovayur**
## Description

This PR adds logging of port-forwarding processes and storing them on the gcp storage among other artifacts and logs.
This is necessary to continue CI flake investigation when the port forwarding process is aborted for an unknown reason.

## Checklist
- [x] Investigated and inspected CI test results
- ~[ ] Unit test and regression tests added~
- ~[ ] Evaluated and added CHANGELOG entry if required~
- ~[ ] Determined and documented upgrade steps~
- ~[ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))~

## Testing Performed
Tested on CI that logs are uploaded correctly for the jobs where they are used. 
Also checked that QA jobs are not affected by the change.

Screenshot 2023-01-11 at 13 05 14
 
image

### CODE REVIEW

File: tests/e2e/lib.sh [hunk 1]
```diff
@@ -239,8 +240,12 @@ start_port_forwards_for_test() {
     ulimit -n 65535 || true
 
     central_pod="$(kubectl -n stackrox get po -lapp=central -oname | head -n 1)"
+
+    mkdir -p "$PORT_FORWARD_LOGS"
+
     for target_port in 8080 8081 8082 8443 8444 8445 8446 8447 8448; do
-        nohup kubectl -n stackrox port-forward "${central_pod}" "$((target_port + 10000)):${target_port}" </dev/null &>/dev/null &
+        log_file="$PORT_FORWARD_LOGS/central-${target_port}.log"
+        nohup kubectl -n stackrox port-forward "${central_pod}" "$((target_port + 10000)):${target_port}" > "${log_file}" 2>&1 &
```
Comment (porridge) [inline tests/e2e/lib.sh:248]:
```suggestion
        nohup kubectl -n stackrox port-forward "${central_pod}" "$((target_port + 10000)):${target_port}" < /dev/null > "${log_file}" 2>&1 &
```
Or was there a reason to stop redirecting stdin from /dev/null?

**Review by porridge — APPROVED** [review]
Nice, just one nitpick.

**Review by gavin-stackrox — APPROVED** [review]

**Pull request merged**